test/integration/e2e_test.go

```diff
@@ -174,7 +174,6 @@ func TestE2EFullIntegration_Browser(t *testing.T) {
 			"--concierge-authenticator-type", "jwt",
 			"--concierge-authenticator-name", authenticator.Name,
 			"--oidc-skip-browser",
-			"--oidc-ca-bundle", federationDomainCABundlePath,
```

Comment:
These lines never should have been in these e2e integration tests and are actually what hid this missing feature from us. The OIDC CA bundle should be auto-discovered by "pinniped get kubeconfig" in these tests.

Reply:
I really like this change - we should seek to make thing as easy and autoconfigurable as possible, and this change reinforces that by reducing the number of CLI arguments.